Plots/Animations/NCL_animate_1.py

```diff
@@ -43,7 +43,7 @@
 ax.coastlines(linewidths=0.5)
 ax.set_extent([-180, 180, -90, 90], ccrs.PlateCarree())
 
-# Use geocat.viz.util convenience function to set axes limits & tick values
+# Use geocat.viz.util convenience function to set axis limits & tick values
```

Comment:
Axes is actually the correct version here, as the util function corrects multiple axes.

```suggestion
# Use geocat.viz.util convenience function to set axes limits & tick values
```

Reply:
Yes, but the phrase "axes limits" is incorrect grammatically.  Maybe change it to "tick values, and limits for axes"?

Reply:
So I think both of you are right to some extent. Grammatically, yes it doesn't make sense, however, in matplotlib axes is an object, not the plural of axis as it would be used in math. That being said, I think the correct way to fix this sentence is to add "the" before axes which would make the sentence "Use geocat.viz.util convenience function to set the axes limits & tick values". I think this helps to show that axes is a singular object instead of the inferred plural of axis.

Reply:
At the risk of sounding pedantic, I still think axes is grammatically correct? I'm totally open to somebody explaining it differently, but the convenience function is setting a lower and upper limit for both the x and y axis, so with multiple axes and multiple limits being set per axis, isn't "axes limits" grammatically fine?

Reply:
Additionally, the util function itself is called ```set_axes_limits_and_ticks```, which did not originate with me.

Reply:
I see what you're saying Anissa. I wasn't reading the sentence with that in mind, I was reading it in terms of when we set the ax variable for the matplotlib axes. But what you pointed out also makes sense here, the function is being used for multiple axes so it would be fine grammatically as well. I think the confusion really does come down to the word choice of matplotlib for making axes mean the whole plot that also contains an x and y axis that are unrelated to the 'axes' name/object.